docs/running_ringpop.md

```diff
@@ -0,0 +1,84 @@
+## Monitoring
```

Comment:
Let's maintain only one copy of this doc. Source of truth for the docs is in the Node.js version.